plan.md

```
@@ -0,0 +1,839 @@
+| Stay current        | Low        | Technical debt, no improvements        |
+| Mastra RAG only     | Low        | Minimal change, easy to revert         |
+| mem0 for semantic   | Medium     | New dependency, LLM costs, migration   |
+| mem0 full migration | High       | Major refactor, working memory changes |
+
+---
+
+## Recommendation
+
+### TL;DR
+
+**Recommended: Full mem0ai migration (replace BOTH working memory AND semantic memory)**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)